db/schema.rb

```diff
@@ -465,6 +465,12 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "picture"
+    t.boolean  "payed",                default: false
+    t.boolean  "swags",                default: false
+    t.boolean  "swags_received"
+    t.string   "company_address"
+    t.string   "vat_registration"
```

Comment:
just vat is enough, doesn't help us to create very long names that are not helping us understand more about what this field is. Then we just create big names, which are potentially less easy to remember.